identity-mdoc/src/jvmTest/kotlin/com/android/identity/mdoc/vical/VicalGeneratorTest.kt

```diff
@@ -0,0 +1,119 @@
+package com.android.identity.mdoc.vical
+
+import com.android.identity.crypto.Crypto
+import com.android.identity.crypto.EcCurve
+import com.android.identity.crypto.EcPrivateKey
+import com.android.identity.crypto.X509Cert
+import com.android.identity.crypto.X509CertChain
+import com.android.identity.crypto.X509CertificateCreateOption
+import com.android.identity.crypto.create
+import kotlinx.datetime.Clock
+import kotlin.test.Test
+import kotlin.test.assertContentEquals
+import kotlin.test.assertEquals
+import kotlin.time.Duration.Companion.days
+import kotlin.time.Duration.Companion.minutes
+import kotlin.time.Duration.Companion.seconds
+
+class VicalGeneratorTest {
+
+    private fun createSelfsignedCert(
+        key: EcPrivateKey,
+        subject: String
+    ): X509Cert {
+        val validFrom = Clock.System.now() - 10.minutes
+        val validUntil = validFrom + 10.minutes
```

Comment:
validFrom is: now - 10m
validUntil is: validFrom + 10m = (now - 10m) + 10m = now.
Seems like, depending on how the check is done and how much time passes before validUntil is checked, Clock.System.now() could become > validUntil very easily, which might make these tests flaky. ...if any tests start checking the validFrom/validUntil times.

Was this intended to be:
validFrom = now - 10m
validUntil = now + 10m
?

Reply:
I don't think we ever test it against `Clock.System.now()` but, yes, this is confusing. I'll change it to what you suggest.